.travis.yml

```diff
@@ -39,6 +39,7 @@ before_install:
     else
       export HOST_OS="$TRAVIS_OS_NAME";
     fi
+  - npm cache clean && npm install --global npm@5
```

Comment:
This is more of a debug commit where this is coming from – I'm trying to see if there's an issue with the old cache, because there are really weird post-install script errors from some step in `build/ci/install.sh`